test/fluree/server/integration/basic_transaction_test.clj

```diff
@@ -11,9 +11,9 @@
   (testing "can create a new ledger w/ JSON"
     (let [ledger-name (str "create-endpoint-" (random-uuid))
           req         (json/write-value-as-string
-                        {"f:ledger" ledger-name
-                         "@context" ["" {"foo" "http://foobar.com/"}]
-                         "@graph"   [{"id"      "ex:create-test"
+                        {"ledger" ledger-name
+                         "@context" ["https://ns.flur.ee" {"foo" "http://foobar.com/"}]
```

Comment:
I think this should be in the conn default context if we're going to need it in every transaction. I think it would make more sense to keep its `f` alias in there, though, to keep that namespace cleaner.

Reply:
I've updated the default conn default context to include it, but users will still need to specify at least the `https://ns.flur.ee` as an explicit `@context` value in every transaction, at least for now. That way credentials will work and we can always expand the transaction without having to check the conn default context.

I do think there's a path towards not including it, it just isn't implemented yet.